tests/node-only/ed25519-key-der.spec.js

```diff
@@ -32,7 +34,7 @@ describe('Ed25519 Keys', () => {
   describe('Ed25519 Private Key', () => {
     describe('DER encoding', () => {
       it('works properly', async () => {
-        const forgeDer = privateKeyDerEncode({privateKeyBytes});
+        const forgeDer = _privateKeyDerEncode({privateKeyBytes});
         const forgeDerBytesBase64 = Buffer.from(forgeDer).toString('base64');
         forgeDerBytesBase64.should.equal(targetPrivateDerBytesBase64);
```

Comment:
We don't use `forge` anymore (this PR removes it); we should clean up the names here to remove confusion.

Reply:
should be addressed here and thanks for catching this: https://github.com/digitalbazaar/ed25519-verification-key-2018/pull/11/commits/6bd0203e3c87fc45648d0707c1f83b21a7878d9f